CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-### 2.1.1 (Next)
```

Comment:
I think this should just be 2.2.0 because we put back a feature. 

A ruby dependency increment isn't exactly a breaking API change.